docs/reference/mapping/types/geo-shape.asciidoc

```diff
@@ -257,57 +252,40 @@ POST /example/_doc
 }
 --------------------------------------------------
 
-*IMPORTANT NOTE:* WKT does not enforce a specific order for vertices thus
-ambiguous polygons around the dateline and poles are possible.
-https://tools.ietf.org/html/rfc7946#section-3.1.6[GeoJSON] mandates that the
-outer polygon must be counterclockwise and interior shapes must be clockwise,
-which agrees with the Open Geospatial Consortium (OGC)
-https://www.opengeospatial.org/standards/sfa[Simple Feature Access]
-specification for vertex ordering.
+[discrete]
+[[polygon-orientation]]
+===== Polygon orientation
 
-Elasticsearch accepts both clockwise and counterclockwise polygons if they
-appear not to cross the dateline (i.e. they cross less than 180° of longitude),
-but for polygons that do cross the dateline (or for other polygons wider than
-180°) Elasticsearch requires the vertex ordering to comply with the OGC and
-GeoJSON specifications. Otherwise, an unintended polygon may be created and
-unexpected query/filter results will be returned.
+A polygon's orientation indicates the order of its vertices: `RIGHT`
+(counterclockwise) or `LEFT` (clockwise).
 
-The following provides an example of an ambiguous polygon. Elasticsearch will
-apply the GeoJSON standard to eliminate ambiguity resulting in a polygon that
-crosses the dateline.
+{es} only uses a polygon's orientation to determine if it crosses the
+international dateline (+/-180° longitude). If so, {es} splits the polygon at
+the dateline. If the difference between a polygon's minimum longitude and the
+maximum longitude is less than 180°, its orientation is not considered and has
+no effect.
```

Comment:
There is something here that I found difficult to read. I tried the following:

```
Elasticsearch only uses a polygon’s orientation to determine if it crosses the international dateline (+/-180° longitude). 
If the difference between a polygon’s minimum longitude and the maximum longitude is less than 180°, the polygon is considered that is not crossing it
and the orientation has no effect. Otherwise the orientation is checked and if the polygon orientation is different to the orientation defined in the mapping
then Elasticsearch considers that the polygon crosses the dateline and the polygon is split across it.
```

Sure can be improve but I think it is nice to provide the two branches of the logic to make it clearer? WDYT?

Reply:
Thanks for the feedback @iverase. I agree. It's clearer to separate the two branches.

I incorporated your feedback with 3396fe4984c37964e788cf3e85ca8e228982fd9c. I tried to preserve most of it. However, I did expand in a few places for clarity.

Thanks for the iteration here. 🙇

Reply:
Yes, that reads nicely.